index.html

```diff
@@ -0,0 +1 @@
+<h1>Hello world!</h1>
```

Comment:
add empty lane at the end of file.
```suggestion
<h1>Hello world!</h1>

```